KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyPresenter.swift

```diff
@@ -263,8 +278,18 @@ final class KarhooLoyaltyPresenter: LoyaltyPresenter {
         }
     }
     
-    private func updateUIWithError(message: String) {
-        view?.showError(withMessage: message)
+    private func updateWithError(_ error: LoyaltyError) {
+        switch error {
+        case .insufficientBalance:
+            internalDelegate?.updateWith(errorMessage: UITexts.Errors.insufficientBalanceForLoyaltyBurning)
+            
```

Comment:
Vertical whitespaces should not be used to separate switch cases, since this code is all about the same. `case` keyword should be enough to keep code organized.

Reply:
Fixed in error handling branch